app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ChargesController < ApplicationController
+  def new
+  end
+
+  def create
+    # Amount in cents
+    @amount = 500
+
+    customer = Stripe::Customer.create(
+      :email => params[:stripeEmail],
+      :source  => params[:stripeToken]
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Operator `=>` should be surrounded by a single space.